spec/features/conversations/index_spec.rb

```diff
@@ -20,8 +20,8 @@
 
     describe 'deleting' do
       before do
-        # delete button
-        click_link class: 'delete'
+        check 'conversation_ids[]'
+        click_button 'Delete Selected'
```

Comment:
needs an `expect` to actually test something here.

Reply:
Thanks. There were already 3 expectations that occurred after this before block. I added a couple additional specs for coverage that was missing.